### PR TITLE
Completed Irish translation of duckduckGo.po

### PR DESCRIPTION
**Pull request by seananocoistin**
I finished the Irish translation of DuckduckGo. It is ready to use.

### CODE REVIEW

File: locales/ga_IE/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -4089,13 +4178,13 @@ msgid "Turn off:"
 msgstr "Múch:"
 
 msgid "Twitter"
-msgstr ""
+msgstr "Twitter"
 
 msgid "Type %sDuckDuckGo%s in the first %sform field"
-msgstr ""
+msgstr "Clóscríobh %sDuckDuckGo%s i réimse an chéad foirme"
```
Comment (jbarrett) [inline locales/ga_IE/LC_MESSAGES/duckduckgo.po]:
```suggestion
msgstr "Clóscríobh %sDuckDuckGo%s i réimse an %schéad foirme"
```

**Comment by seananocoistin** [comment]
It would be great to have it implemented as soon as possible. From: John BarrettSent: Dé Céadaoin 8 Eanáir 2020 19:52To: duckduckgo/duckduckgo-localesCc: Seanán Ó Coistín; MentionSubject: Re: [duckduckgo/duckduckgo-locales] Completed Irish translation of duckduckGo.po (#230) @jbarrett requested changes on this pull request.This is fantastic, many thanks @seananocoistin! 👍I think the suggested change will get the tests rolling.In locales/ga_IE/LC_MESSAGES/duckduckgo.po:>    msgid "Type %sDuckDuckGo%s in the first %sform field" -msgstr "" +msgstr "Clóscríobh %sDuckDuckGo%s i réimse an chéad foirme" ⬇️ Suggested change -msgstr "Clóscríobh %sDuckDuckGo%s i réimse an chéad foirme" +msgstr "Clóscríobh %sDuckDuckGo%s i réimse an %schéad foirme" —You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by jbarrett** [comment]
:+1: Many thanks @seananocoistin!

**Comment by seananocoistin** [comment]
Can I take this to mean that the Irish translation will now be uploaded and implemented on DuckDuckGo.com ? From: John BarrettSent: Déardaoin 9 Eanáir 2020 16:33To: duckduckgo/duckduckgo-localesCc: Seanán Ó Coistín; MentionSubject: Re: [duckduckgo/duckduckgo-locales] Completed Irish translation of duckduckGo.po (#230) 👍 Many thanks @seananocoistin!—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by jbarrett** [comment]
> Can I take this to mean that the Irish translation will now be uploaded and implemented on DuckDuckGo.com?

Yes, it should be live now - thanks!

**Pull request merged**